src/components/customNode.tsx

```diff
@@ -0,0 +1,47 @@
+import { Handle, Position } from "reactflow";
+import "reactflow/dist/style.css";
+import Star from "@/components/icons/Star";
+import classNames from "classnames";
+import { getColorByAccess } from "@/utils/Helper";
+
+export function CustomNode({ data, id }: any) {
```

Comment:
This component can just be name `NodeCard`

Reply:
fixed